configs/ot2_defconfig

```diff
@@ -91,7 +85,29 @@ BR2_PACKAGE_DROPBEAR=y
 BR2_PACKAGE_IW=y
 BR2_PACKAGE_NETWORK_MANAGER=y
 BR2_PACKAGE_NGINX=y
-BR2_PACKAGE_NGINX_HTTP_SSL_MODULE=y
+# BR2_PACKAGE_NGINX_HTTP_CHARSET_MODULE is not set
```

Comment:
huh, stuff like this usually doesn't make it into a defconfig, wonder if nginx is doing something weird to parse these. would expect them to be `BR2_PACKAGE_NGINX_HTTP_GZIP_MODULE=n` and so on

Reply:
Yeah, it is weird and mysterious.